public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+openapi: 3.0.0
+info:
+  title: CTC Guarantee Balance
+  contact: {}
+  version: '2.0'
```

Comment:
Missing `info.description` - see updated guidelines for Info section in [Confluence](https://confluence.tools.tax.service.gov.uk/display/DTRG/OpenAPI+Specification+structure+and+style).

Suggest reusing description in v1.0 YAML file:
```
description: |
    # Overview
    Use the CTC Guarantee Balance API to provide traders with up-to-date
    information about how much of their guarantee funds they have left to
    use when organising transit movements.
```

Reply:
Ah, I'd missed this as my branch isn't quite up to date -- so I got the older version. Cool, will sort this (and other bits) out.